### PR TITLE
MCR-2359 (3) MCRConfigurableInstance bugfix generator init

### PR DESCRIPTION
**Pull request by sebhofmann**
[Link to jira](https://mycore.atlassian.net/browse/MCR-2359).

### CODE REVIEW

File: mycore-pi/src/main/java/org/mycore/pi/MCRGenericPIGenerator.java [hunk 1]
```diff
@@ -143,7 +144,11 @@
 
     public MCRGenericPIGenerator() {
```
Comment (rsteph-de) [inline mycore-pi/src/main/java/org/mycore/pi/MCRGenericPIGenerator.java:145]: Do you really need the "empty" default constructor?

**Review by rsteph-de — COMMENTED** [review]
see comments

File: mycore-pi/src/main/java/org/mycore/pi/MCRGenericPIGenerator.java (hunk 1 above)
Comment (sebhofmann) [reply mycore-pi/src/main/java/org/mycore/pi/MCRGenericPIGenerator.java:145]: Yes because there is a non public constructor for testing.

**Review by rsteph-de — APPROVED** [review]
looks good for me

**Pull request merged**